claim/README.md

```diff
@@ -167,11 +167,11 @@ Use these keys and the information below to troubleshoot the ACLK.
 
 If `cloud-enabled` is `false`, you probably ran the installer with `--disable-cloud` option.
 
-Additionally, check that the `netdata cloud` setting in `netdata.conf` is set to `enable`:
+Additionally, check that the `enabled` setting in `var/lib/netdata/cloud.d/cloud.conf` is set to `true`:
 
 ```ini
-[general]
-    netadata cloud = enable
+[global]
+    enabled = true
```

Comment:
At least for me, when I see only `enabled` inside `[global]` section, I have the first impression that we are disable Netdata, considering that `global` applies to the whole system (Here I am doing a comparison with other sections that we have ). I think we could use the previous variable, or to use another that creates an explicit relationship with the cloud.

Reply:
This is the `[global]` section of `cloud.conf`, not `netdata.conf`. It's a completely different configuration space. Also, it's part of the requirements, so I cannot put anything back into `netdata.conf`.

Reply:
I notice this later, but the README was the third file, so I had in my mind that we were working yet with `netdata.conf'. Thank you!